mergify_engine/gh_pr_fullifier.py

```diff
@@ -160,11 +161,23 @@ def compute_travis_url(pull, **extra):
     )["url"]
 
 
-def compute_weight_and_status(pull, **extra):
+def disabled_by_rules(pull, **extra):
     labels = [l.name for l in pull.labels]
     if extra["branch_rule"]["disabling_label"] in labels:
+        return "Disabled with label"
+
+    pull_files = [f.filename for f in pull.get_files()]
+    for w in extra["branch_rule"]["disabling_files"]:
+        filtered = fnmatch.filter(pull_files, w)
+        if filtered:
+            return "Disabled, %s have been changed" % filtered[0]
```

Comment:
`"Disabled, %s is modified in this pull request"`

sounds clearer if somebody does not know mergify.